com.sap.cloud.lm.sl.cf.core/src/test/resources/com/sap/cloud/lm/sl/cf/core/cf/detect/deployed-components-10.json

```diff
@@ -12,9 +12,9 @@
           "services": [],
           "providedDependencyNames": [],
           "deployAttributes": {
-            "success-marker": "SUCCESS",
```

Comment:
Why are these reordered? Is this something to do with the type of the map? Is it HashMap, because if it is this ordering might not work on my machine. :)

Reply:
Adapter uses TreeMap so it re-order entries in the Map. So it should not be different on any machine